hub/components/RealmHeader/index.tsx

```diff
@@ -11,13 +11,17 @@ import OverflowMenuHorizontalIcon from '@carbon/icons-react/lib/OverflowMenuHori
 import ProgressBarRound from '@carbon/icons-react/lib/ProgressBarRound';
 import UserFollow from '@carbon/icons-react/lib/UserFollow';
 import WalletIcon from '@carbon/icons-react/lib/Wallet';
+
 import * as NavigationMenu from '@radix-ui/react-navigation-menu';
+import { useWallet } from '@solana/wallet-adapter-react';
 import type { PublicKey } from '@solana/web3.js';
 import { pipe } from 'fp-ts/lib/function';
 import Link from 'next/link';
-import { useMemo } from 'react';
+import { useCallback, useMemo } from 'react';
 import { useMediaQuery } from 'react-responsive';
 
+import useWalletStore from 'stores/useWalletStore';
```

Comment:
to get the `connection`, please use `https://github.com/solana-labs/governance-ui/blob/main/hub/hooks/useCluster.ts` instead:

```ts
import { useCluster } from '@hub/hooks/useCluster';

const [cluster] = useCluster();
const connection = cluster.connection;
```

Reply:
certainly!